# 1

Известно, что генеральная совокупность распределена нормально со средним квадратическим отклонением, равным 16. Найти доверительный интервал для оценки математического ожидания a с надежностью 0.95, если выборочная средняя M = 80, а объем выборки n = 256.

80 +- 1.96*16/16

In [3]:
80+1.96

81.96

In [4]:
80-1.96

78.04

доверительный интервал [78.04;81.96]

# 2

В результате 10 независимых измерений некоторой величины X, выполненных с одинаковой точностью, получены опытные данные: 6.9, 6.1, 6.2, 6.8, 7.5, 6.3, 6.4, 6.9, 6.7, 6.1 Предполагая, что результаты измерений подчинены нормальному закону распределения вероятностей, оценить истинное значение величины X при помощи доверительного интервала, покрывающего это значение с доверительной вероятностью 0,95. 

In [6]:
import numpy as np

In [5]:
x=[6.9, 6.1, 6.2, 6.8, 7.5, 6.3, 6.4, 6.9, 6.7, 6.1]

In [10]:
s=np.std(x,ddof=1)

In [17]:
b=np.mean(x)

In [13]:
b+2.262*s/np.sqrt(10)

6.912461744087575

In [14]:
b-2.262*s/np.sqrt(10)

6.267538255912426

доверительный интервал [6.27;6.91]

# 3

Утверждается, что шарики для подшипников, изготовленные автоматическим станком, имеют средний диаметр 17 мм.
Используя односторонний критерий с α=0,05, проверить эту гипотезу, если в выборке из n=100 шариков средний диаметр
оказался равным 17.5 мм, а дисперсия известна и равна 4 кв.мм.

Н0 = 17 мм
Н1 = 17.5 мм

Zt = 1.645

In [19]:
(17.5-17)/(2/10)

2.5

Zr>Zt

In [ ]:
Альтернативная гипотеза верна при уровне значимости 5%

# 4

Продавец утверждает, что средний вес пачки печенья составляет 200 г.
Из партии извлечена выборка из 10 пачек. Вес каждой пачки составляет:
202, 203, 199, 197, 195, 201, 200, 204, 194, 190.
Известно, что их веса распределены нормально.
Верно ли утверждение продавца, если учитывать, что доверительная вероятность равна 99%?

In [ ]:
M не равно M0
a = 1%
t критерий

In [20]:
x1=[202, 203, 199, 197, 195, 201, 200, 204, 194, 190]

In [21]:
s1=np.std(x1,ddof=1)

In [22]:
b1=np.mean(x1)

In [ ]:
t = 3.250

In [23]:
(b1-200)/(s1/np.sqrt(10))

-1.0651074037450896

In [ ]:
t расчетное по модулю меньше t расчетного
Гипотеза Но верна. 